<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumeSegmentationWidget</class>
 <widget class="QWidget" name="VolumeSegmentationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VolumeSegmentationWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <row>
          <property name="text">
           <string>Seed</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>X</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Y</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Z</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="intensityProbBtn">
         <property name="text">
          <string>Load Intensity Probs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="textureProbBtn">
         <property name="text">
          <string>Load Texture Probs</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="probabilityBtn">
       <property name="text">
        <string>Compute Probability</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="segmentBtn">
       <property name="text">
        <string>Segment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="newSeedBtn">
       <property name="text">
        <string>Choose New Seed</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveBtn">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>intensityProbBtn</sender>
   <signal>clicked()</signal>
   <receiver>VolumeSegmentationWidget</receiver>
   <slot>loadIntensityProbs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textureProbBtn</sender>
   <signal>clicked()</signal>
   <receiver>VolumeSegmentationWidget</receiver>
   <slot>loadTextureProbs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>26</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>probabilityBtn</sender>
   <signal>clicked()</signal>
   <receiver>VolumeSegmentationWidget</receiver>
   <slot>computeProbability()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>segmentBtn</sender>
   <signal>clicked()</signal>
   <receiver>VolumeSegmentationWidget</receiver>
   <slot>segment()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newSeedBtn</sender>
   <signal>clicked()</signal>
   <receiver>VolumeSegmentationWidget</receiver>
   <slot>newSeed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveBtn</sender>
   <signal>clicked()</signal>
   <receiver>VolumeSegmentationWidget</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadIntensityProbs()</slot>
  <slot>loadTextureProbs()</slot>
  <slot>computeProbability()</slot>
  <slot>segment()</slot>
  <slot>save()</slot>
  <slot>newSeed()</slot>
 </slots>
</ui>
